hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -881,7 +867,7 @@ def descendants():
             + self.nodes[ancestor.ir_end :]
         )
 
-    def lower_common_block_offset(self):
+    def lower_common_node_offset(self):
         """Sometimes we find ourselves in a situation where changes to one part
         of the byte stream unlock changes to other parts. Sometimes this is
```

Comment:
Here, and below, we probably want to replace "byte stream" with "choice sequence" or some other phrase.

Reply:
oof yeah, the comments need a big pass here in general. I'll probably leave that as an item in the issue...don't want to make reviewing this harder than it already is.